cli/cmd/root.go

```diff
@@ -337,4 +340,5 @@ func addProxyConfigFlags(cmd *cobra.Command, options *proxyConfigOptions) {
 	cmd.PersistentFlags().UintSliceVar(&options.ignoreInboundPorts, "skip-inbound-ports", options.ignoreInboundPorts, "Ports that should skip the proxy and send directly to the application")
 	cmd.PersistentFlags().UintSliceVar(&options.ignoreOutboundPorts, "skip-outbound-ports", options.ignoreOutboundPorts, "Outbound ports that should skip the proxy")
 	cmd.PersistentFlags().BoolVar(&options.disableExternalProfiles, "disable-external-profiles", options.disableExternalProfiles, "Disables service profiles for non-Kubernetes services")
+	cmd.PersistentFlags().BoolVar(&options.noInitContainer, "no-init-container", options.noInitContainer, "Experimental: Omit the proxy-init container when injecting the proxy; requires the linkerd-cni plugin to already be installed")
```

Comment:
I talked this over with @grampelberg, and I think we should call this flag "--linkerd-cni-enabled", since the init container is more of an internal detail of what's happening. Can you change this to:
```suggestion
	cmd.PersistentFlags().BoolVar(&options.noInitContainer, "linkerd-cni-enabled", options.noInitContainer, "Experimental: Allow the linkerd-cni plugin to handle routing traffic to the proxy; requires that the linkerd-cni plugin is installed first with 'linkerd install-cni'")
```

Reply:
Done.